app/views/view-application.njk

```diff
@@ -27,6 +28,7 @@
       {% if withdrawConfirmationForm %}
         {{ viewWithdrawConfirmationForm(applicationId, page, crumb) }}
       {% endif %}
+      {{ viewClaimConfirmationForm(applicationId, page, crumb) }}
```

Comment:
The placement of this form doesn't tie in with what is shown in [User Story 148784](https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/148784)